### DISCUSSION TITLE
Missing information (unable to figure out how to simply open up a pdf file to start annotating)

### DISCUSSION THREAD

**Discussion by Somnia-terra** [RyotaUshio/obsidian-pdf-plus#261] [Q&A] [answered]
Reviewing this plugin page on the instructions. I tried finding information on how to open up a .pdf file. The getting started seems to be missing this very important information. Or I am not interpreting the instructions correctly. I have the plugin installed and enabled on Obsidian but cannot find a button or a function to open a .pdf file on a local device to start annotating. Trying to understand the intuition of the plugin. I tried searching on a mainstream search engine and searching on GitHub to get an answer but can't seem to get an answer for what I want.

Or maybe there is something wrong with it? Not sure.

Extra info:
I am using Obsidian v1.6.7 and installed the plugin via the plugin browser in Obsidian. Running Ubuntu 24.04 LTS.

**Comment by RyotaUshio** [comment, marked as answer]
Whether you have this plugin installed or not, if you have a PDF file **in your vault**, you can just open it from File Explorer (the left sidebar) or from Quick Switcher (Ctrl+O).

**Reply by Somnia-terra** [reply]
Ohhhhh I see. Yes, this makes much more sense now. I am very new to Obsidian. Thank you, I appreciate your response and that helped a lot to get started.
